Engine/src/Engine/ResourceManagement/Serializer.cpp

```diff
@@ -127,6 +127,7 @@ namespace Engine
 
 		std::cout << sceneJson << std::endl;
 		//TODO: Bind serialization to GUI event once we have one.
+		ResourceManager::getInstance()->saveJsonFile(sceneJson, sceneFile);
```

Comment:
Not a request to change but FYI @JackyLau01 mentioned interest in changing the way these two functions behave (by returning json's and receiving it as input, instead of making use of the Resource Manager. May want to check with them to avoid future headaches.

Reply:
Good to know, thanks.